FSharp.AutoComplete/Program.fs

```diff
@@ -617,6 +641,36 @@ module internal Main =
 
               main state
 
+          | SymbolUse ->
+              let symboluses =
+                  async {
+                      let! symboluse = tyRes.GetSymbol(line, col, lineStr)
+                      if symboluse.IsNone then return None else
+                      let! symboluses = tyRes.GetUsesOfSymbolInFile symboluse.Value.Symbol
+                      return Some {
+                        Name = symboluse.Value.Symbol.DisplayName
+                        Uses =
+                          [ for su in symboluses do
+                              yield { StartLine = su.RangeAlternate.StartLine
+                                      StartColumn = su.RangeAlternate.StartColumn + 1
+                                      EndLine = su.RangeAlternate.EndLine
+                                      EndColumn = su.RangeAlternate.EndColumn + 1
```

Comment:
I'm not sure how to fix this. FCS seem to report 0-based columns ([lines are 1-based](https://github.com/fsharp/FSharp.Compiler.Service/blob/251c00058cb4ec48e9fc31a7a814d8a3f4898d3f/src/fsharp/vs/service.fsi#L136)). Do you want me to only add 1 to `StartColumn`? That doesn't seem right to me. That would leave `testva` highlighted.